Firmware/MVP/battsci.cpp

```diff
@@ -185,28 +362,28 @@ void BATTSCI_sendFrames()
       if( (LTC68042result_hiCellVoltage_get() < 42000 ) && (LTC68042result_loCellVoltage_get() > 29500) )
       { //all cells are within acceptable range
         frameSum_AA += BATTSCI_writeByte( 0x00 );                                         //enable assist & regen
-      
+
       } else if( (LTC68042result_hiCellVoltage_get() > 42000 ) && (LTC68042result_loCellVoltage_get() < 29500))
       { //at least one cell is over-changed AND at least one cell is under-charged
         //JTS2doLater: set P-code... battery is beyond redeemable.
         frameSum_AA += BATTSCI_writeByte( 0x30 );                                         //b00110000 disables regen & assist
 
       } else if(LTC68042result_hiCellVoltage_get() > 42000) //42000 = 4.2000
       { //at least one cell is overcharged.  disable regen
-        frameSum_AA += BATTSCI_writeByte( 0x20 );                                         //b00100000 disables regen  
-      
+        frameSum_AA += BATTSCI_writeByte( 0x20 );                                         //b00100000 disables regen
+
       } else if(LTC68042result_loCellVoltage_get() < 29500) //29500 = 2.9500 volts
       { //at least one cell is undercharged. disable assist
         frameSum_AA += BATTSCI_writeByte( 0x10 );                                         //b00010000 disables assist
       }
 
       frameSum_AA += BATTSCI_writeByte( 0x40 );                                           //Charge request.  0x20=charge@4bars, 0x00=charge@background
-      frameSum_AA += BATTSCI_writeByte( 0x61 );                                           //Never changes
+      frameSum_AA += BATTSCI_writeByte( 0x61 );                                           //Never changes	0x00 N	0x01 P1440	0x60 N	0x32 N
```

Comment:
I believe this is the BCM hardware revision.  Odd that different values would cause P1440...